Problems when use new support library for iOS

Hi,

Im taking error:

```
/Users/runner/work/ezored/ezored/files/modules/support-lib/djinni/objc/DJIError.h:23:55: error: unknown type name 'NSString'
[[noreturn]] void throwUnimplemented(const char *ctx, NSString *msg);
                                                      ^
1 error generated.
```

Full log:
https://github.com/ezored/ezored/runs/3747388613?check_suite_focus=true#step:13:403

Can anyone help me with this?

--- a/djinni/objc/DJIError.mm
+++ b/djinni/objc/DJIError.mm
@@ -13,10 +13,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-
+// clang-format off
 #include <Foundation/Foundation.h>
 #include "DJIError.h"
 #include <exception>
+// clang-format on
 static_assert(__has_feature(objc_arc), "Djinni requires ARC to be enabled for this file");
 
 namespace djinni {

